sddi-base/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN set -ex && \
   ls -lah /wheels
 
 # ckanext-grouphierarchy ######################################################
-ARG CKANEXT_SDDI_VERSION="1.0.1"
+ARG CKANEXT_SDDI_VERSION="bugfix-admin-config"
```

Comment:
If it is ready, make a release in the Upstream repo and put the new version here.
There are a lot of new features, we need a minor version step 1.1.0.

Reply:
It is changed